src/main/resources/META-INF/sql/mysql-schema.sql

```diff
@@ -116,3 +116,10 @@ CREATE TABLE IF NOT EXISTS consumers_stats (
   metadata text
 )ENGINE=InnoDB CHARACTER SET utf8;
 
+CREATE TABLE IF NOT EXISTS tokens (
```

Comment:
I think we can drop the mysql schema file at all

Reply:
I am not sure, because some users often use MySQL database, I think if this file does not violate Apache protocol, it can give users a reference.